pug/list_testTypes.pug

```diff
@@ -1,123 +1,132 @@
-extend default
 
+extend default
 include common.pug
 
+
 block vars
-  - var page_title = "Test Types";
-  
+  - var page_title = "All Test Types";
+
+
 block content
   .container-fluid
     pre #{''}
     pre #{''}
   
     if(permissions.userHas(user, "forms:edit"))
-      a.NewTestType.btn.btn-outline-primary(href = '#', data-toggle = "modal" data-target = "#NewTestTypeModal") Create New Test Type Form
+      a(href = '#', data-toggle = "modal" data-target = "#NewTestTypeModal").NewTestType.btn.btn-warning
+        img.small-icon(src = '/images/edit_icon.svg')
+        | &nbsp; Create New Test Type Form
     
       #NewTestTypeModal.modal.fade(tabindex = '-1' role = 'dialog' aria-labelledby = 'NewTestTypeLabel' aria-hidden = 'true')
         .modal-dialog(role = "document")
           .modal-content
             .modal-header
-              h5#NewTestTypeLabel.modal-title New Test Type
+              h5#NewTestTypeLabel.modal-title Create New Test Type
               button.close(type = "button", data-dismiss = "modal", aria-label = "Close")
                 span(aria-hidden = true) &times;
             .modal-body
               div
                 | Enter the new test type's form ID.
                 br
-                | NOTE: this ID should have only underscores and alphanumerics - the human-readable name will be set on the next page.
-                input.NewTestTypeName.form-control(type = "text")
+                | NOTE: this ID should have only underscores and alphanumerics (no spaces) - the human-readable name will be set on the next page.
+              pre #{''}
+              input.NewTestTypeName.form-control(type = "text")
             .modal-footer
-              button.btn.btn-secondary(type = "button", data-dismiss = "modal") Close
-              button.btn.btn-primary.NewTestTypeButton(type = "button") Create
+              button.btn.btn-danger(type = "button", data-dismiss = "modal") Close
+              button.btn.btn-success.NewTestTypeButton(type = "button") Create
+      
       script.
         $('button.NewTestTypeButton').click(function()
         {
-          window.location.href = "/tests/" + $('button.NewTestTypeButton').parents('.modal-content').find('input').val() + "/new";
+          window.location.href = '/tests/' + $('button.NewTestTypeButton').parents('.modal-content').find('input').val() + '/new';
```

Comment:
This is curious… most modern web frameworks have client-side routing, but not pug?

Reply:
[sent you an email about this, since I'd like to close this PR and move onto the next one]